spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AboutController.java

```diff
@@ -101,14 +98,21 @@ public class AboutController {
 
 	private DataflowMetricsProperties dataflowMetricsProperties;
 
+	private ObjectProvider<GitInfoContributor> gitInfoContributor;
+
+	private ObjectProvider<BuildInfoContributor> buildInfoContributor;
+
 	public AboutController(StreamDeployer streamDeployer, LauncherRepository launcherRepository, FeaturesProperties featuresProperties,
-			VersionInfoProperties versionInfoProperties, SecurityStateBean securityStateBean, DataflowMetricsProperties monitoringProperties) {
+			VersionInfoProperties versionInfoProperties, SecurityStateBean securityStateBean, DataflowMetricsProperties monitoringProperties,
```

Comment:
Even though nobody is probably creating their own AboutController, technically this is a breaking API change. To be safe and to keep in good habits lets instead mark the current constructor deprecated and then create a new one that also includes the 2 new fields. Wdyt?

Reply:
Perfect! done!